cypress/integration/dib-travel/my-travels/my-travels.spec.ts

```diff
@@ -0,0 +1,128 @@
+describe('My travels', () => {
+  beforeEach(() => {
+    cy.login();
+
+    cy.get('dib-navbar .home-page').contains(' My Travels ').click();
+  });
+
+  it('should display "My Travels" in the navbar', () => {
+    cy.get('dib-navbar .navbar__content').contains(' add trip ').should('exist');
+  });
+
+  it('should displays message that there are no active travels', () => {
+    cy.get('app-my-travels dib-travels-list .table-container.empty-list .empty-trips-section').should(
+      'contain',
+      " You don't have any active travels, but you can always create one "
+    );
+  });
+
+  it('should displays button to create new travel', () => {
+    cy.get('app-my-travels dib-travels-list .table-container.empty-list .dib-button-new').should(
+      'contain',
+      'Create new travel '
+    );
+  });
+
+  it('should check search for past travels', () => {
+    cy.get('app-my-travels .mat-tab-links').contains(' Past ').click();
+
+    cy.get('app-my-travels dib-travels-list dib-input input').type('CYQA');
+
+    cy.get('app-my-travels dib-travels-list .name').should('have.text', 'CYQA Bot bussines trip');
+
+    cy.get('app-my-travels dib-travels-list dib-input input').clear().type('123456789');
+
+    cy.get('app-my-travels dib-travels-list .empty-trips-section').should(
+      'have.text',
+      ' There are no past travels to be shown currently '
+    );
+  });
+
+  it('should check sorting by price', () => {
+    cy.get('app-my-travels .mat-tab-links').contains(' Past ').click();
+
+    cy.get('app-my-travels dib-travels-list .clickable').contains(' Price ').click();
+
+    cy.get('app-my-travels dib-travels-list  .uppercase').eq(1).should('contain', '2', '764,82');
+
+    cy.get('app-my-travels dib-travels-list .clickable').contains(' Price ').click();
+
+    cy.get('app-my-travels dib-travels-list  .uppercase').eq(1).should('not.contain', '2 764,82');
+  });
+
+  it('should check sorting by date', () => {
+    cy.get('app-my-travels .mat-tab-links').contains(' Past ').click();
+
+    cy.get('app-my-travels dib-travels-list .uppercase').eq(0).should('contain', ' 2 Apr 2021 ');
+
+    cy.get('app-my-travels dib-travels-list .clickable').contains(' Travel date ').click();
+
+    cy.get('app-my-travels dib-travels-list .uppercase').eq(0).should('not.contain', ' 2 Apr 2021 ');
+  });
+
+  it('should open view more section', () => {
+    cy.get('app-my-travels .mat-tab-links').contains(' Past ').click();
+
+    cy.get('app-my-travels dib-travels-list .item .name')
+      .contains('sub_981ba1a0-f9cc-48f0-9644-1343d6e70929')
```

Comment:
The same as above, you can also call this 'sub_981ba1a0-f9cc-48f0-9644-1343d6e70929' from JSON file and maybe for this test case, you can use 'CYQA Bot business trip' instead of this.